Add TopupAccountActivity and use future kinsdk

berryve opened the pull request.

Commit:
Add TopupAccountActivity and use future kinsdk

shayb commented on app/src/main/java/org/kinecosystem/kinit/blockchain/Wallet.kt:
```diff
@@ -87,6 +87,7 @@ class Wallet(context: Context, dataStoreProvider: DataStoreProvider,
     private val providerUrl = if (type == Type.Main) SDK_MAIN_NETWORK_URL else SDK_TEST_NETWORK_URL
     private val networkId = if (type == Type.Main) SDK_MAIN_NETWORK_ID else SDK_TEST_NETWORK_ID
     private val walletCacheName = if (type == Type.Test) TEST_NET_WALLET_CACHE_NAME else MAIN_NET_WALLET_CACHE_NAME
+    val topupManager: TopupManager
```
if you can just put it on the top section with the other val

shayb commented on app/src/main/java/org/kinecosystem/kinit/blockchain/TopupManager.kt:
```diff
@@ -0,0 +1,75 @@
+
+                            val srcAppId: String = if (srcPublicAddress == masterAccount.publicAddress)
+                                masterAppId
+                            else kinitAccountData.data()["__link_$srcPublicAddress"] ?: ""
```
move to method
and extend to . account

Commit:
code review improvements

Commit:
Replace with real topup activity using base from transfer module
update to latest sdk
improve structure
still missing the animation and correct success screen
